### PR TITLE
Assign dev-flow-team as reviewer if no reviewer assigned

### PR DESCRIPTION
**Pull request by lilithxxx**
Assign dev-flow-team as reviewer if no reviewer is assigned and the PR has been opened for more than 24 hours.

### CODE REVIEW

**Comment by lilithxxx** [comment]
@apb7 PTAL. Thanks

File: lib/assignReviewers.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+module.exports.assignReviewers = async function(context) {
+  pullRequestsPromiseObj = await context.github.pullRequests.getAll(
+    context.repo({per_page: 40}));
```
Comment (apb7) [inline lib/assignReviewers.js]: Hmm, this is looking good but I see that a number of steps are common for the merge conflicts check and this check. Let's merge the two checks and put them in a single file. In that way, we will be able to reduce the number of requests made. What are your views?

Comment (lilithxxx) [reply lib/assignReviewers.js]: Yea that sounds better. I am on it

Comment (lilithxxx) [reply lib/assignReviewers.js]: Done

**Comment by lilithxxx** [comment]
@apb7 PTAL

**Comment by apb7** [comment]
Hi Rishav, a couple of questions:

1. What steps did you take to manually test this?
2. Can you please add tests for this functionality?

Thanks!

**Comment by lilithxxx** [comment]
Hi @seanlip, now that every file has a code owner, every PR will have a reviewer assigned to him/her after opening the PR. Hence, I can't understand the purpose of this PR. It doesn't need to assign a reviewer as every file has a code owner. What are your opinions? Thanks

**Comment by seanlip** [comment]
That's a very good question! I would ask: is there ever a chance, now or in the future, that some file does not have a code owner?

I think we are going for the answer "no". But if that's the case, then we need to guarantee that every file has a code owner and that this state of affairs is maintained in perpetuity. I don't think those checks exist yet.

If you had to choose between writing those checks and writing this PR, I think writing those checks will have a bigger impact. But until we have those checks, this PR is still relevant.

**Comment by lilithxxx** [comment]
@seanlip so should I write those checks instead and close this PR or should I continue with this only? Thanks

**Comment by seanlip** [comment]
That is a decision for the dev workflow team (which probably means you) :)

**Comment by lilithxxx** [comment]
Ok I am gonna write those checks instead as I also think that would have a greater impact. Closing this PR!

**Comment by seanlip** [comment]
Sounds good! Though you might want to liaise with @YashJipkate: https://github.com/oppia/oppia/issues/6321

**Pull request closed without merging**